src/main/java/com/puppycrawl/tools/checkstyle/checks/TodoCommentCheck.java

```diff
@@ -50,13 +50,26 @@
  * &lt;module name="TodoComment"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * //TODO: ..... violation
```

Comment:
Please add some java statement in front , at least `i++; // TODO: do differently in future`. This Check is only applicable to java files, as have to be placed under Treewalker module.

Please add example of Todo comment that default config will not catch like ` // todo: haha.   // Ok

Reply:
Ok i will change it and add example for OK state